public/delete_thread.php

```diff
@@ -0,0 +1,23 @@
+<?php 
+
+	require('database.php');
+
+	session_start();
+
+	$postId = $_POST['id'];
+
+	$authorId = $_SESSION['user_id'];
+
+	$db = new Database();
+
+	$threadsQuery = "DELETE FROM threads WHERE id= $postId AND author_id= $authorId";
+
+	$postsQuery = "DELETE FROM posts WHERE id= $postId AND author_id= $authorId";
```

Comment:
Good critical thinking to realize you need to delete all the posts from a thread when you delete the thread.

There's an issue with this query though – it's not going to delete the correct posts.

Let's say you have a Thread with ID 5 which has Posts with IDs 8 and 9. You also have another Thread with ID 8 which has Posts with IDs 17 and 18.

What happens if you try to delete thread 8? According to these queries, it will delete Thread 8 and Post 8. No bueno. :)

What you want to do is to delete all posts associated with Thread 8. How would you do that? Hint: what column in the posts table *associates* it with a Thread?